Source/WebCore/platform/graphics/gbm/DMABufFormat.h

```diff
@@ -30,6 +30,10 @@
 #include <cstddef>
 #include <cstdint>
 
+namespace IPC {
+template<typename T, typename U> struct ArgumentCoder;
+}
+
```

Comment:
You can simply include `<wtf/ArgumentCoder.h>` instead, but I don't think you need this, read below.